lib/before-cleanApp.js

```diff
@@ -1,8 +1,11 @@
 const { cleanSnapshotArtefacts } = require("../snapshot/android/project-snapshot-generator");
 const { isAndroid } = require("../projectHelpers");
+const { setProcessInitDirectory } = require("./utils");
 
 module.exports = function (hookArgs) {
     if (isAndroid(hookArgs.platformInfo.platform)) {
-        cleanSnapshotArtefacts(hookArgs.platformInfo.projectData.projectDir);
+        const projectDir = hookArgs.platformInfo.projectData.projectDir;
+        setProcessInitDirectory(projectDir);
```

Comment:
Shouldn't this be outside of `if` statement? In case when command is executed for `iOS` platform `setProcessInitDirectory` method will not be called.

Reply:
In this case we will not execute any logic, so there's no need to reset the variable. I prefer setting it only when we will execute some work.